app/cdap/components/Replicator/constants.tsx

```diff
@@ -23,3 +23,8 @@ export const PROGRAM_INFO = {
   programType: 'workers',
   programId: 'DeltaWorker',
 };
+
+export enum ErrorType {
+  generic = 'generic',
```

Comment:
is this generic type being used?

Reply:
nope, it's not used as of now. Removed it.